reportportal_client/model/response.py

```diff
@@ -1,12 +1,15 @@
 import json
 
+from ..errors import EntryCreatedError, OperationCompletionError
```

Comment:
No, it is recommended way to use relative imports. Moreover, other modules also use relative import as well.